content/W2015/W15-event-profsdinner.md

```diff
@@ -0,0 +1,23 @@
+Title: Dinner With Profs
+Date: 2015-03-26 5:30PM
+Category: Events
+Tags: profs, dinner
+Slug: dinner-with-profs
+Author: Anna Lorimer
+Summary: WiCS and WiM are hosting a Dinner with Profs event! Come get to know other successful women in your field. 
+
+WiCS and WiM would like to invite all undergraduate ciswomen and transwomen, and non-binary folk to our Dinner with Profs event!
+
+This event is an opportunity for ciswomen and transwomen, and non-binaryfolk to in Math and CS to enjoy a semi-formal dinner while getting to know other successful people in the same fields.
+It promises to be a great event! 
```

Comment:
Another pro tip is to make sure it's not leaving whitespace at the end of newlines; the vim line wrap shortcut does this by default, so I have to go back and clean that up (which is kinda annoying but c'est la vie)